docs/workflow/requirements/linux-requirements.md

```diff
@@ -1,65 +1,100 @@
-Requirements to build dotnet/runtime on Linux
-======================
+# Requirements to build dotnet/runtime on Linux
 
-This guide will walk you through the requirements to build dotnet/runtime on Linux. Before building there is environment setup that needs to happen to pull in all the dependencies required by the build. There are two suggested ways to go about doing this. First you are able to use the Docker environments provided by https://github.com/dotnet/dotnet-buildtools-prereqs-docker, or you can set up the environment yourself. The documentation will go over both ways of building. Using Docker allows you to leverage our existing images which already have an environment set up.
+* [Docker](#docker)
+* [Environment](#environment)
+  * [Toolchain Setup](#toolchain-setup)
+    * [Additional Requirements for Cross-Building](#additional-requirements-for-cross-building)
+      * [Alpine ARM Cross-Building Requirements](#alpine-arm-cross-building-requirements)
+  * [Unsupported OSes](#unsupported-oses)
 
-General instructions for building are [here](../README.md).
-Instructions for building CoreCLR for Linux are [here](../building/coreclr/linux-instructions.md).
+This guide will walk you through the requirements to build _dotnet/runtime_ on Linux. Before building there is environment setup that needs to happen to pull in all the dependencies required by the build.
 
+There are two suggested ways to go about doing this. The first one is you are able to use the [provided Docker environments](https://github.com/dotnet/dotnet-buildtools-prereqs-docker) used in the official builds, or you can set up the environment yourself. The documentation will go over both ways. Using Docker allows you to leverage our existing images which already have an environment set up, while using your own environment grants you better flexibility on having other tools at hand you might need.
 
-Docker
-==================
-
-Install Docker; see https://docs.docker.com/install/.
+## Docker
 
+Install Docker. For further installation instructions, see [here](https://docs.docker.com/install/). You can find the official .NET images in [their Docker hub](https://hub.docker.com/_/microsoft-dotnet). You can also use the images used for the official builds. More details on these ones in the [Linux building instructions doc](/docs/workflow/building/coreclr/linux-instructions.md#docker-images).
```

Comment:
> You can find the official .NET images in [their Docker hub](https://hub.docker.com/_/microsoft-dotnet).

The official .NET images on Docker hub are not relevant for building dotnet/runtime repo.